docs_source_files/content/webdriver/web_element.zh-cn.md

```diff
@@ -386,3 +433,109 @@ To achieve this, the parent WebElement is chained with 'findElements' to access
   }
   {{< / code-panel >}}
 {{< / code-tab >}}
+
+## Get Active Element
+
+It is used to track (or) find DOM element which has the focus in the current browsing context.
+
+{{< code-tab >}}
+  {{< code-panel language="java" >}}
+  import org.openqa.selenium.*;
+  import org.openqa.selenium.chrome.ChromeDriver;
+  
+  public class activeElementTest {
+    public static void main(String[] args) {
+      WebDriver driver = new ChromeDriver();
+      try {
+        driver.get("http://www.google.com");
+        driver.findElement(By.cssSelector("[name='q']")).sendKeys("webElement");
+  
+        // Get attribute of current active element
+        String attr = driver.switchTo().activeElement().getAttribute("title");
+        System.out.println(attr);
+      } finally {
+        driver.quit();
+      }
+    }
+  }
+  {{< / code-panel >}}
+  {{< code-panel language="python" >}}
+  from selenium import webdriver
+  
+  driver = webdriver.Chrome()
+  driver.get("https://www.google.com")
+  driver.find_element_by_css_selector('[name="q"]').send_keys("webElement")
+  
+  # Get attribute of current active element
+  attr = driver.switch_to.active_element.get_attribute("title")
+  print attr
+  {{< / code-panel >}}
+  {{< code-panel language="csharp" >}}
+  using OpenQA.Selenium;
+  using OpenQA.Selenium.Chrome;
+
+  namespace ActiveElement {
+   class ActiveElement {
+    public static void Main(string[] args) {
+     IWebDriver driver = new ChromeDriver();
+     try {
+      // Navigate to Url
+      driver.Navigate().GoToUrl("https://www.google.com");
+      driver.FindElement(By.CssSelector("[name='q']")).SendKeys("webElement");
+
+      // Get attribute of current active element
+      string attr = driver.SwitchTo().ActiveElement().GetAttribute("title");
+      System.Console.WriteLine(attr);
+     } finally {
+      driver.Quit();
+     }
+    }
+   }
+  }
+  {{< / code-panel >}}
+  {{< code-panel language="ruby" >}}
+  require 'selenium-webdriver'
+  driver = Selenium::WebDriver.for :chrome
+  begin
+    driver.get 'https://www.google.com'
+    driver.find_element(:css,'[name="q"]').send_keys("webElement")
```

Comment:
```ruby
driver.find_element(css: '[name="q"]').send_keys('webElement')
```
- Pass a Hash instead of two parameters 
- Consistent string literal quoting style, in this case using single quotes.

Just my opinion of course :+1:

Reply:
Hi @Madh93,

We welcome suggestions.

Update code sample as suggested.